### PR TITLE
drtprod: tpcc_init script fix

### PR DESCRIPTION
**Pull request by vidit-bhat**
Ran some manual tests to verify

Epic: none
Release note: None

### CODE REVIEW

File: pkg/cmd/drtprod/scripts/tpcc_init.sh [hunk 1]
```diff
@@ -20,12 +20,14 @@ if [ -z "${WORKLOAD_CLUSTER}" ]; then
 fi
 
 # script is responsible for importing the tpcc database for workload
-roachprod ssh "${WORKLOAD_CLUSTER}":1 -- "tee tpcc_init.sh > /dev/null << EOF
+roachprod ssh "${WORKLOAD_CLUSTER}":1 -- "tee tpcc_init.sh > /dev/null << 'EOF'
+TPCC_DB=cct_tpcc
```
Comment (nameisbhaskar) [inline pkg/cmd/drtprod/scripts/tpcc_init.sh:24]: We need not extract it in an environment variable as it si used in only one place. But, I am fine either ways. Please do test that the generated script works before merging. Thanks

**Review by nameisbhaskar — APPROVED** [review]
Overall looks good. Please do check if the generated scripts execute correctly. Thanks!

File: pkg/cmd/drtprod/scripts/tpcc_init.sh (hunk 1 above)
Comment (vidit-bhat) [reply pkg/cmd/drtprod/scripts/tpcc_init.sh:24]: Already tested

**Comment by vidit-bhat** [comment]
TFTR!
bors r=nameisbhaskar

**Pull request merged**